Dockerfile

```diff
@@ -32,7 +32,7 @@ RUN ln -s /usr/lib64/libgcrypt.so.11.8.2 /usr/lib64/libgcrypt.so.11
 LABEL name="HPE CSI Driver for Kubernetes" \
       maintainer="HPE Storage" \
       vendor="HPE" \
-      version="2.4.0-beta" \
+      version="2.4.0-beta2" \
```

Comment:
do we need to name beta2 ?

Reply:
If we decide to push another chart we need new tags.